### PR TITLE
First pass at an IATI status blogpost

### PR DESCRIPTION
**Pull request by andylolz**

### CODE REVIEW

**Review by matmaxgeds — COMMENTED** [review]
Cool :) - could consider including (for those less technical readers!):
1. when you could use it e.g. if something is not displaying, or looks out of date
2. what to do if something says 'failed' e.g. contact the 'owner'
3. the benefit e.g. is automated, get news of issues before they become problems

**Comment by andylolz** [comment]
All great suggestions, @matmaxgeds! Thanks. I’ll have a go at incorporating at some point this weekend.

File: _posts/2020-08-14-iati-status.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: A status dashboard for IATI services
+comments: true
+author: andy
+excerpt: Welcome to Code for IATI! Learn more about what we're trying to do and how you can get involved.
```
Comment (markbrough) [inline _posts/2020-08-14-iati-status.md]: Excerpt needs to be edited!

File: _posts/2020-08-14-iati-status.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+These incidents are unpredictable, and can cause disruption for users. So it’s common for online services to provide a status dashboard, to automatically alert users of current issues.
+
+[Facebook has one](https://developers.facebook.com/status/dashboard/). [Twitter has one](https://api.twitterstat.us/). [Dropbox has one](https://status.dropbox.com/). [Github has one](https://www.githubstatus.com/). In fact: check it yourself. Think of an online service, and then do a search for their status dashboard. Chances are, you’ll probably find one.
+
+IATI provides a number of services, but there’s no central status dashboard. So we had a go at making one.
+
+It’s here:
+[https://status.codeforiati.org/](https://status.codeforiati.org/)
+
+It’s based on [open source code](https://github.com/IATI/IATI-Website-Tests) created by the IATI secretariat. This code runs automated tests on various IATI services, to check they are performing as expected. For example:
```
Comment (markbrough) [inline _posts/2020-08-14-iati-status.md]: suggest: "created by the IATI Secretariat's technical team a number of years ago"

File: _posts/2020-08-14-iati-status.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+it checks the dashboard appears to have been updated recently
+It checks the validator correctly identifies some invalid XML as such
+It checks the datastore returns CSV when requested
+…and lots more things besides
+
+We took this code, fixed a couple of bugs, scheduled it to run daily, and added [a public website](https://status.codeforiati.org/) so anyone can see the current status. We also added [an RSS feed](https://status.codeforiati.org/feed.xml), and a badge that shows the status right now:
+
+![Current status](https://status.codeforiati.org/status.svg)
+
+It’s easy enough to extend with new checks. For example, d-portal tests are minimal, and there are no tests in place for the new datastore or new validator. We think this is a fundamental service that should be provided by a web service provider. We’d be pleased if the IATI secretariat adopted this.
```
Comment (markbrough) [inline _posts/2020-08-14-iati-status.md]: suggest:

"We think this is a fundamental service that should be provided by all service providers, to inform users of any issues in using different services. We’d be very happy for the IATI Secretariat to adopt this, if they (and their users) find it useful!"

File: _posts/2020-08-14-iati-status.md (hunk 1 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: ha! Thanks, good spot

File: _posts/2020-08-14-iati-status.md (hunk 3 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: Ah, much better!

File: _posts/2020-08-14-iati-status.md (hunk 2 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: Yes good point. Possibly worth linking to this: https://github.com/IATI/IATI-Website-Tests/graphs/contributors

File: _posts/2020-08-14-iati-status.md (hunk 1 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: Fixed!

File: _posts/2020-08-14-iati-status.md (hunk 2 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: Fixed!

File: _posts/2020-08-14-iati-status.md (hunk 3 above)
Comment (andylolz) [reply _posts/2020-08-14-iati-status.md]: Fixed!

**Review by markbrough — APPROVED** [review]
Looks great to me!!

**Comment by andylolz** [comment]
Thanks both! 🌟

**Pull request merged**